src/libraries/Microsoft.PowerFx.Core/Entities/External/IExternalDataSource.cs

```diff
@@ -21,6 +21,8 @@ internal interface IExternalDataSource : IExternalEntity, IExternalPageableSymbo
 
         bool IsWritable { get; }
 
+        bool IsClearable { get; }
```

Comment:
@LucGenetier  - this is what caused your PR failure.  Just rebase on latest to pick this up. 
https://github.com/microsoft/Power-Fx/pull/2425/checks?check_run_id=25337498835